stable/aws-calico/templates/daemon-set.yaml

```diff
@@ -0,0 +1,137 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: calico-node
+  labels:
+    k8s-app: calico-node
+{{ include "aws-calico.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: calico-node
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-node
+    spec:
+      priorityClassName: system-node-critical
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      hostNetwork: true
+      serviceAccountName: calico-node
```

Comment:
The service account should be composed using a Helm function, see https://github.com/aws/eks-charts/blob/master/stable/appmesh-controller/templates/deployment.yaml#L22

Reply:
changed